src/crimson/osd/pg_backend.h

```diff
@@ -45,7 +45,6 @@ class PGBackend
 					   const ec_profile_t& ec_profile);
 
   using read_errorator = ll_read_errorator::extend<
-    crimson::ct_error::input_output_error,
```

Comment:
read_errorator is changed in futurized store, crimson::ct_error::input_output_error is add in futurized store since bluestore may return this error. so remove it here.